src/Utils/LocationUtils.cs

```diff
@@ -66,7 +66,7 @@ public static Vec3f GetRelativeSpeakerLocation(EntityPos speakerPos, EntityPos l
         public static Vec3f GetRelativeSpeakerLocation(Vec3f speakerPos, EntityPos listenerPos)
         {
             var relativeSpeakerCoords = speakerPos - listenerPos.XYZFloat;
-            var listenerHeadAngle = listenerPos.Yaw + listenerPos.HeadYaw - Math.PI / 2;
+            var listenerHeadAngle = listenerPos.Yaw + listenerPos.HeadYaw - Math.PI;
```

Comment:
Pretty sure that's an incorrect fix and now listener's head is rotated 180 degrees. 
While OpenAL may(or may not) treat sounds directly from the back the same way as sounds directly from the front, it **will** break our other positional code, so things like doppler effect, muffling behind walls or cases of rapid movement/high ping will produce artifacts or work in an unexpected way.
```suggestion
            var listenerHeadAngle = listenerPos.Yaw + listenerPos.HeadYaw;
```

Reply:
Discussed in discord, after 1.20 the game measures Yaw from the South due to that being "mathematical north"(Z+) and the Z axis being inverted, so this change is correct